### DISCUSSION TITLE
No video files in SavedClips and SentryClips

### DISCUSSION THREAD

**Discussion by iamdooser** [marcone/teslausb#835] [Q&A]
Hello, I appreciate anyone who might be able to direct me with this; and perhaps suggest any logs I should post.

When viewing from the web interface, there are timestamped folders inside the SavedClips and SentryClips folders, each with a event.json file and thumb.png, but no video files. (There are .mp4 files in the folders in RecentClips, so that appears to be working correctly). (And therefore, the archive server reflects the same...no video files).

**Comment by sockmonkey0223** [comment]
I've just noticed this behavior as well. I have a 2018 Model 3 and 2023 Model Y and both are doing the same thing. I hadn't reviewed the uploaded footage in a while so I see this issue goes back to 4/4/24 which is the last time I reviewed.

**Comment by sockmonkey0223** [comment]
I've checked one of the Tesla USB devices and see that the content there matches what was copied to the NAS (folders with event.json & thumb.png but no videos). However, I also see that there are old folders with videos in them that I know had been previously copied to the NAS - I think the sync process should have cleaned those up.
I'm guessing either the file system is full or something is wrong with the file system (or both). Just strange that this would happen with two different Tesla USB installs.

**Comment by DMBlakeley** [comment]
This post prompted me to check my recordings also. I found pretty much the same. Last full recording was on March 18th and after that only event.json & thumb.png files. Found the same looking at the viewer on the Tesla screen (2023 Model Y). The TeslaUSB had worked without issue for months so I had never checked.

**Comment by sockmonkey0223** [comment]
After writing this, I found the following issue: https://github.com/marcone/teslausb/issues/825

Sounds like there _may_ be a fix (last few comments) but I haven't tried it yet.
